service/policy/db/query.sql

```diff
@@ -230,6 +230,75 @@ WHERE
 GROUP BY
     ad.id, an.id, nfq.fqn, n_grants.grants;
 
+---------------------------------------------------------------- 
+-- ATTRIBUTE VALUES
+----------------------------------------------------------------
+
+-- name: ListAttributeValues :many
+
+SELECT
+    av.id,
+    av.value,
+    av.active,
+    JSON_STRIP_NULLS(JSON_BUILD_OBJECT('labels', av.metadata -> 'labels', 'created_at', av.created_at, 'updated_at', av.updated_at)) as metadata,
+    av.attribute_definition_id,
+    fqns.fqn
+FROM attribute_values av
+LEFT JOIN attribute_fqns fqns ON av.id = fqns.value_id
+WHERE (
+    (sqlc.narg('active')::BOOLEAN IS NULL OR av.active = sqlc.narg('active')) AND
+    (NULLIF(@attribute_definition_id, '') IS NULL OR av.attribute_definition_id = @attribute_definition_id::UUID)
+)
+GROUP BY av.id, fqns.fqn;
+
+-- name: GetAttributeValue :one
+SELECT
+    av.id,
+    av.value,
+    av.active,
+    JSON_STRIP_NULLS(JSON_BUILD_OBJECT('labels', av.metadata -> 'labels', 'created_at', av.created_at, 'updated_at', av.updated_at)) as metadata,
+    av.attribute_definition_id,
+    fqns.fqn,
+    JSONB_AGG(
+        DISTINCT JSONB_BUILD_OBJECT(
+            'id', kas.id,
+            'uri', kas.uri,
+            'public_key', kas.public_key
+        )
+    ) FILTER (WHERE avkag.attribute_value_id IS NOT NULL) AS grants
+FROM attribute_values av
+LEFT JOIN attribute_fqns fqns ON av.id = fqns.value_id
+LEFT JOIN attribute_value_key_access_grants avkag ON av.id = avkag.attribute_value_id
+LEFT JOIN key_access_servers kas ON avkag.key_access_server_id = kas.id
+WHERE av.id = $1
+GROUP BY av.id, fqns.fqn;
+
+-- name: CreateAttributeValue :one
+INSERT INTO attribute_values (attribute_definition_id, value, metadata)
+VALUES (@attribute_definition_id, LOWER(@value), @metadata)
+RETURNING id, value,
```

Comment:
I am guessing we have this, but can you please ensure we have an integration test that validates creation of a value normalizes to lower case?

Reply:
Or we could also create a new integration tests issue to collect this need and any others if you prefer to split out the work from this task.

Reply:
Verified that we already have one: https://github.com/opentdf/platform/blob/main/service/integration/attribute_values_test.go#L142-L158